### DISCUSSION TITLE
[Bug] Text vanishes when viewing file on mobile with screen rotated to landscape

### DISCUSSION THREAD

**Discussion by TomNicholas** [community/community#33539] [Mobile]
Hi, I keep encountering the same bug when viewing `.txt` files on mobile. Basically it displays the text in the file fine until you rotate the screen with auto-rotate into landscape orientation, at which point all the text disappears! The line numbers are still there, but the text doesn't even return if you rotate back to portrait view. 

I think I see the issue more often when there are text lines longer than the width of the screen too, but I'm not sure about that. This happens every time with certain files anyway.

This is pretty annoying - I want to use landscape mode quite often to see all of longer lines of text, but I can't do that at all with this bug.

I originally thought this was due to my janky old phone, but I recently got a new Pixel 6 Pro and I still see the same issue (on Android 13).

Happy to look up any further specifics if that would diagnose the issue.

**Comment by eliperkins** [comment]
Hey @TomNicholas ! Thanks for this feedback. 🙇  I can see how disappearing text would be frustrating, especially when rotating to see longer lines of code. I'd be frustrated too.

Some good news for you though: the awesome Android engineering team fixed this bug within the app last week, and the fix should be rolling out to production in v1.81.0 of GitHub for Android today! If you want to get this fix sooner, you can join the beta group for GitHub for Android on the Play Store and help us verify this bug is fixed in the public beta.

Let me know if this bug persists after updating to GitHub for Android v1.81.0.

**Reply by TomNicholas** [reply]
Amazing! I'll try the new version and let you know if it doesn't fix the issue 🙂
